CHANGELOG.md

```diff
@@ -50,6 +50,16 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - The removal of `/v1/report` APIs in this release, that was communicated in release 4.0.0, has been postponed by one release. Consequently, the `/v1/report` APIs will continue to be available in this release.
 - The `/api/docs/swagger` API previously required read on the resource `Integration`.
   Now it only requires users to be authenticated to via the API docs.
+- ROX-18173: A few public endpoints will soon require authentication, ensure that any flow interacting with these endpoints is authenticated going forward:
+  - `/v1/availableauthproviders`
+  - `/v1/config/public`
+  - `/v1/featureflags`
+  - `/v1/internal/dev/certcheck`
+  - `/v1/internal/dev/envvars`
+  - `/v1/internal/dev/random`
+  - `/v1/internal/dev/reconciliationstats`
+  - `/v1/internal/dev/replicate`
```

Comment:
I'm not sure about these ones. They are actually not part of the release build. Let's ditch them for now.